packages/tldraw/src/lib/tools/SelectTool/childStates/EditingShape.ts

```diff
@@ -21,6 +21,21 @@ export class EditingShape extends StateNode {
 	hitShapeForPointerUp: TLShape | null = null
 	private info = {} as EditingShapeInfo
 
+	private isTextInputFocused(): boolean {
```

Comment:
Do we have a better way to know this?

Reply:
Though this is actually something we should fix (i.e. whether a shape should prevent keyboard shortcuts while editing). We used to have a check on whether the user was focused on text inputs or text areas...